src/views/listview/list-view.component.js

```diff
@@ -123,7 +130,10 @@ angular.module('patternfly.views').component('pfListView', {
       // update the actions based on the current item
       ctrl.updateActions(item);
 
-      $window.requestAnimationFrame(function () {
+      // Hide the kebab until we determine dropup or dropdown to avoid flicker
+      ctrl.kebabMenuStyle = {visibility: 'hidden'};
```

Comment:
General question... do we have or use a "visibility: hidden" CSS class in core PF, or was using "ng-class" a no go since we're trying to simply hide the refresh/load tweak?

Reply:
there is an invisible class in bootstrap I could use. Good thought, I'll update.